.env.example

```
@@ -1,5 +1,16 @@
+# App
 PORT=3333
 HOST=0.0.0.0
 NODE_ENV=development
-APP_KEY=73rHnSEgVPLfFLpTk8omwzBrMk-c75rX
+APP_KEY=   # preencha com uma chave
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)